transport/src/main/java/io/netty/bootstrap/AbstractBootstrap.java

```diff
@@ -417,17 +394,18 @@ final ChannelHandler handler() {
     }
 
     final Map<ChannelOption<?>, Object> options() {
-        return copiedMap(options);
+        return Collections.unmodifiableMap(options);
     }
 
     final Map<AttributeKey<?>, Object> attrs() {
-        return copiedMap(attrs);
+        return Collections.unmodifiableMap(attrs);
```

Comment:
same comment, and same applies to the equiv `child` methods in `ServerBootstrap`